src/styles/card.scss

```
@@ -0,0 +1,82 @@
+@import "utils/variables";
+
+.card {
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid $grey;
+  background-color: $white;
+
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    padding: 0 3px;
+    height: 134px;
+    max-width: 100%;
+    margin-bottom: 40px;
+    vertical-align: middle;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: $secondary;
+    font-size: 10px;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+  }
+
+  &__text {
+    color: $secondary;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__button {
+    display: block;
+    padding: 11px 69px 11px 68px;
```

Instead of using such big values as 68px you can use 11px for all sizes and add width: 100% to a button
```suggestion
    width: 100%;
    padding: 11px;
```